test/data/mini_example/explanation.fasta

```diff
@@ -1,23 +1,28 @@
 >ref
-CGCCCATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCG
+CGCCCATGCA ACTAGCGATGCTAGCTAGCTAGCT TACGACTGGCCATGCG
 GCGGCAT ATTTATAACGGGC ATACATAAGGGGCTCATCGATCGATTTCGG
 ATCGGGGGGCCCCCATTTTAAACGG   GGATCTTGACTCTGGAAAACTTTTA
 ACGCCGGGAATCGGTAGTCCTTTCGCGGGG ATATATTT TACGGGATATAA
 CGATCGGATCGGATCGATAATAGGTCTCGGTTGCCAACTGATCGTACCAA
 ATATTTCTGCGGGGCT   CGCCCCTCCGCGATTA AGAG TCGGCTAACGGTT C
 ACCCATGCGGATTATAAACGGG ATATTAAGGGCTTT ATATTAAGGGCTTT
-AGGGCTAGCTAGGTCCAA
+AGGGCTAGCTAGGTCCAA   AGCTTTCGGAACGTAC GGTAACGTGTA CGGTA
+CGTACTGAAAAAAACCAAACGGGGAAATATTAACCGGCCTTTGCCTGGGA
+CGCTGTGACTGTACGGGGGGGCGGGCCTTATT
 
 >new_Creature
-CGCCCATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCG                                      breakpoints:
-GCGGCAT ATACATAAGGGGCTCATCGATCGATTTCGG                              <- Deletion         ref: [57, 70] deleted
-ATCGGGGGGCCCCCATTTTAAACGG CCCCGGGGCCAATTT GGATCTTGACTCTGGAAAACTTTTA <- Insertion        ref: after 125 something inserted
-ACGCCGGGAATCGGTAGTCCTTTCGCGGGG ATATATTT ATATATTT TACGGGATATAA       <- Duplication      ref: (180 or 189) something inserted
+CGCCCATGCAACTAGCGATGCTAGCTAGCTAGCTTACGACTGGCCATGCG                                          breakpoints:
+GCGGCAT ATACATAAGGGGCTCATCGATCGATTTCGG                              <- Deletion             ref: (57, 70] deleted
```

Comment:
Attention: equivalent to the previous would be `(56` here, so I guess it was mistaken before?